### PR TITLE
use go v1.19 and update dependencies

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
cc @saad-ali

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,16 @@
 module github.com/container-storage-interface/spec
 
-go 1.16
+go 1.19
```
Comment (jdef) [inline go.mod:3]: is this version actually required to build CSI (w/ reasonable deps)? if not, why not bump to 1.17 or 1.18 instead? CSI module doesn't need to be bleeding edge (and thereby force downstream users to use the latest Go)

Comment (humblec) [reply go.mod]: sure @jdef , I have declared it on 1.18 in the last refresh.

**Review by jdef — APPROVED** [review]

**Comment by jdef** [comment]
@saad-ali any opinions here? if not, i'll merge as is

**Comment by saad-ali** [comment]
No objections. LGTM

**Pull request merged**